### ISSUE TITLE
Link prefetch for navbar doesn't work with keyboard focus

### ISSUE THREAD

**Issue by oscartbeaumont** [mattrax/Mattrax#316]
If you focus it via the keyboard the route data is not prefetched. It is prefetched when hovering.

**Comment by oscartbeaumont** [comment]
This applies both to the top bar and the command palette now that it also used link components.
